assets/xml/objects/object_fhg.xml

```diff
@@ -0,0 +1,20 @@
+<Root>
+    <File Name="object_fhg" Segment="6">
+        <!-- <Skeleton Name="gPhantomHorseSkel" Type="Standard" LimbType="Skin" Offset="0xB098"/> -->
+        <Blob Name="gPhantomHorseSkel" Size="8" Offset="0xB098"/>
+
+        <Animation Name="gPhantomHorseAnim_00B4C8" Offset="0xB4C8"/>
+        <Animation Name="gPhantomHorseAnim_00B9D0" Offset="0xB9D0"/>
+        <Animation Name="gPhantomHorseAnim_00C65C" Offset="0xC65C"/>
+        <Animation Name="gPhantomHorseAnim_00CB1C" Offset="0xCB1C"/>
+        <Animation Name="gPhantomHorseAnim_00DDB8" Offset="0xDDB8"/>
+        <Animation Name="gPhantomHorseAnim_00E8A0" Offset="0xE8A0"/>
+        
+        <Texture Name="gPhantomGanonTitleCardTex" Format="i8" Width="128" Height="120" Offset="0x59A0"/>
```

Comment:
Textures in both of these objects should ideally have a filename through `OutName` (it will probably be required by the extraction process eventually)